pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -258,8 +258,13 @@ func createMonitoringResources(engine chaosTypes.EngineInfo, recEngine *reconcil
 	return reconcile.Result{}, nil
 }
 
-// getChaosRunnerENV return the env required for chaos-runner
+//getChaosRunnerENV returns the env required for chaos-runner
 func getChaosRunnerENV(cr *litmuschaosv1alpha1.ChaosEngine, aExList []string, ClientUUID string) []corev1.EnvVar {
+	return getChaosRunnerDefaultENV(cr, aExList, ClientUUID)
+}
+
+// getChaosRunnerDefaultENV return the default env required for chaos-runner
+func getChaosRunnerDefaultENV(cr *litmuschaosv1alpha1.ChaosEngine, aExList []string, ClientUUID string) []corev1.EnvVar {
```

Comment:
any specific reason for creating this fn?

Reply:
Just renaming the functions, as with the earlier approach, one more function was added.